src/pages/Checkout/CheckoutPage.vue

```diff
@@ -920,6 +920,10 @@ export default {
 <style lang="scss">
 @import 'settings';
 
+.basket-list-container {
+	min-height: 600px;
```

Comment:
The min-height will need to be applied to your new upsell container rather than the item list. This list can vary well beyond 600 and would still have the issue of the loan upsell popping in and expanding it's height.